include/FreeRTOS_DNS.h

```diff
@@ -49,17 +49,27 @@
     #endif /* ipconfigBYTE_ORDER == pdFREERTOS_BIG_ENDIAN */
 
     #define ipLLMNR_PORT           5355 /* Standard LLMNR port. */
-    #define ipDNS_PORT             53   /* Standard DNS port. */
-    #define ipDHCP_CLIENT          67
-    #define ipDHCP_SERVER          68
-    #define ipNBNS_PORT            137 /* NetBIOS Name Service. */
-    #define ipNBDGM_PORT           138 /* Datagram Service, not included. */
+
+
+    #if ( ipconfigBYTE_ORDER == pdFREERTOS_BIG_ENDIAN )
+        #define ipMDNS_IP_ADDRESS    0xe00000fbU /* 224.0.0.251 */
+    #else
+        #define ipMDNS_IP_ADDRESS    0xfb0000e0U /* 224.0.0.251 */
```

Comment:
Would a macro for building IP addresses in the native endianness make this more clear?